CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# (WIP) Contributing Info 
+
+## Semantic Commit Messages
+
+Inspired by [Git Commit Msg][git-commit-msg], [Angular Commit Message Format][angular-commit-message-format] and [Conventional Commits][conventional-commits]
+
+
+### <a name="commit-message-format"></a> Commit Message Format
+
+Each commit message consists of a **header**, a **body** and a **footer**. The header has a special
+format that includes a **type**, a **scope**, and a **subject**:
+
+```html
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+> Any line of the commit message cannot be longer 100 characters!<br/>
+This allows the message to be easier to read on GitHub as well as in various Git tools.
+
+##### Type
+
+Must be one of the following:
+
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing tests
+* **chore**: Changes to the auxiliary tools such as release scripts
+* **build**: Changes to the dependencies, devDependencies, or build tooling
+* **ci**: Changes to our Continuous Integration configuration
```

Comment:
Do we need so many different type types? 
"perf" seems to be a subtype of "refactor".
"build" and "ci" may be a subtype of "chore".

So I suggest to remove those subtypes.

Reply:
I don't like 'chore' it's like 'item'. To be a little bit more specific should be helpful if you are searching for a problem.
'chore' should be avoided if you have a better type.

Reply:
Ok, good point. Then we should keep "ci" and "build"
What about "perf"? I belive "perf" would be used very rarely. I can only remember one commit a few years ago where this type could have been applied.

Reply:
I suggest a type **demo** additional to **docs** (it's not the same)
I'm not sure, but **vm* might be useful for the tobago-vm.apache.org stuff.
What about version changes as release preparations? The Jira versions are managed in demo, so we can use **demo**.
Or is **demo** a scope?

Reply:
If you fix an example in the demo, do you write the commit title?
* "demo(popup): fix example"
* "fix(demo): popup example"

Reply:
I'm fine with "fix(demo): popup example". "demo" is more a scope.